app/src/main/kotlin/nl/ndat/tvlauncher/data/resolver/ChannelResolver.kt

```diff
@@ -39,9 +39,12 @@ class ChannelResolver {
 			do {
 				try {
 					val packageName = cursor.getString(PreviewChannel.Columns.COL_PACKAGE_NAME)
-					if (packageName != "com.google.android.tvrecommendations") add(
-						PreviewChannel.fromCursor(cursor).toChannel()
-					)
+					// Check if app has channel.
+					if (!cursor.getString(PreviewChannel.Columns.COL_APP_LINK_INTENT_URI).isNullOrEmpty()){
+						if (packageName != "com.google.android.tvrecommendations") add(
```

Comment:
If you merge these two if statements it should be good to merge 👍

Reply:
Thanks for the advice.
I have merged these if statement.